### PR TITLE
Updated some language etc 

### PR DESCRIPTION
**Pull request by iah**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [X] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [X] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

---

Instant Answer Page:  https://duck.co/ia/view/arch_linux_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @direwolf424

### CODE REVIEW

**Comment by daxtheduck** [comment]
@iah Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
                 "val": "Download a package without installing it"
             }
         ],
-        "Yaourt - A Pacman Frontend": [
+        "Yaourt - A wrapper for the AUR and regular packages.": [
             {
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/arch-linux.json]: Please could you change the entry in the `section_order` array to match this title?

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 2]
```diff
@@ -106,7 +106,7 @@
             }
             
         ],
-        "Linux Command": [
+        "Various Linux commands": [
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/arch-linux.json]: Please could you change the entry in the `section_order` array to match this title?

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 3]
```diff
@@ -75,7 +75,7 @@
                 "val": "Download a package without installing it"
             }
         ],
-        "Yaourt - A Pacman Frontend": [
+        "Yaourt - A wrapper for the AUR and regular packages.": [
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/arch-linux.json]: Please could you change the entry in the `section_order` array to match this title?

**Comment by tagawa** [comment]
Hi @iah. Thank you for these improvements. I've just left a couple of comments that should make the checks pass.

**Comment by iah** [comment]
Sorry for the quick PR. Here's some updates.

File: share/goodie/cheat_sheets/json/arch-linux.json (hunk 3 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/arch-linux.json]: Please remove the trailing period.

File: share/goodie/cheat_sheets/json/arch-linux.json [hunk 4]
```diff
@@ -106,7 +106,30 @@
...
+                "key": "pacman -Rdd xyz",
+                "val": "Remove package 'xyz', which is required by another package, without removing the dependent package"
+            },
+            {
+                "key": "pacman -Rn xyz",
+                "val": "Prevent the creation of the backup configuration files use the -n option"
+            }
+        ],
+
+        "Various Linux commands": [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/arch-linux.json]: `General Commands`

**Comment by tagawa** [comment]
Hi @iah - looks good so let's merge! Thank you for the improvement.

**Pull request merged**